tests/components/google_mail/test_init.py

```diff
@@ -29,7 +29,7 @@ async def test_setup_success(
     await hass.config_entries.async_unload(entries[0].entry_id)
     await hass.async_block_till_done()
 
-    assert not len(hass.services.async_services().get(DOMAIN, {}))
+    assert not hass.services.async_services().get(DOMAIN, {})
```

Comment:
I think this is equivalent:
```suggestion
    assert not hass.services.async_services().get(DOMAIN)
```